prov/verbs/src/ep_rdm/verbs_ep_rdm.c

```diff
@@ -389,20 +392,20 @@ static inline int fi_ibv_rdm_tagged_test_addr(const char *devname,
 	struct rdma_cm_id *test_id;
 	int test = 0;
 	if (rdma_create_id(NULL, &test_id, NULL, RDMA_PS_TCP)) {
-		FI_IBV_ERROR("Failed to create test rdma cm id: %s\n",
+		VERBS_INFO(FI_LOG_CORE, "Failed to create test rdma cm id: %s\n",
```

Comment:
Is the log selection here and below better as FI_LOG_CORE or FI_LOG_AV or FI_LOG_EP_CTRL?